### ISSUE TITLE
The Frames and contents view of the main pane (or designer surface section) of the Inspector window are not accessible using keyboard

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#293]
#### Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.

Create projects:

1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched.

### Repro-Steps:
1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio.
2. Navigate to Main pane (Designer surface) of the inspector window.
3. Navigate to the "Frames and contents view of the main pane (or designer surface section) of the Inspector window
4. Observe the keyboard and behavior on this Main pane (Designer surface) of the inspector window.
### Expected Behavior
Each and every Frames and contents of the main pane (or designer surface section) of the Inspector window should be  accessible using keyboard.
The active keyboard focus should be visible on the Frames and contents view surface.
### Actual Behavior
None of the Frames and contents of the main pane (or designer surface section) of the Inspector window is accessible using keyboard.
Note: The user is able to rotate the frames and contents ( i.e., Designer Surface) using arrow keys.
Note: While keyboard user is rotating the designer surface, the keyboard focus is not available on the same.
### User Impact
The keyboard user might face difficulty to interact  with each and every frames and contents of the Inspector surface.

![kb9_mainpane-content frame_notaccessiblewithkb](https://user-images.githubusercontent.com/22956130/39987607-e0e00aca-5782-11e8-8b12-03e324bbeaad.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Inspector-MAC
Wipro
MAS2.1.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
